src/BootstrapTooltip.js

```diff
@@ -77,14 +77,26 @@ export default declare("BootstrapTooltip.widget.BootstrapTooltip", [_WidgetBase,
         }
 
         if ($targetElement.length > 0) {
-            $targetElement.tooltip({
-                title: () => {
-                    return this._tooltipText;
-                },
-                placement: this.tooltipLocation,
-                trigger: this._getTrigger(),
-                html: this.tooltipRenderHTML
-            });
+            try {
+                $targetElement.tooltip({
+                    title: () => {
+                        return this._tooltipText;
+                    },
+                    placement: this.tooltipLocation,
+                    trigger: this._getTrigger(),
+                    html: this.tooltipRenderHTML
+                });
+            } catch(e) {
+                console.warn(
+                    "Did you configure BootstrapTooltip widget correctly? Couldn't start tooltip methods in the element with class '" +
```

Comment:
Is this the only error that this problem could occur ?

Reply:
This is just a error handling while starting the bootstrap or jquery functions, so it will only prevent the widget to break the whole app if something goes wrong with Jquery version.